coalib/collecting/Importers.py

```diff
@@ -143,6 +143,8 @@ def iimport_objects(file_paths, names=None, types=None, supers=None,
     :param attributes: Attribute(s) an object needs to (all) have
     :param local:      if True: Objects need to be defined in the file they
                        appear in to be collected
+    :param verbose:    Whether verbose logs shall be displayed on console or
+                       not.
```

Comment:
I'm totally cool! No I'm not actually :P everybody keeps pinging me. Can you take over reviewing stuff for @Redridge and @mr-karan ? They need their stuff merged as today is the deadline for that...